enterprise/backend/src/metabase_enterprise/semantic_search/index.clj

```diff
@@ -226,29 +239,83 @@
     (when (seq conditions)
       (into [:and] conditions))))
 
+(defn- keyword-search-query
+  "Build a keyword search query using postgres full-text search."
+  [index search-context]
+  (let [filters (search-filters search-context)
+        search-string (:search-string search-context)
+        ts-search-vector (str/replace (postgres-spec/to-tsquery-expr search-string) "'" "")
```

Comment:
this producing results like 'cat':* rather than cat:* so i hacked around this by removing the `'`